packages/tools/kolibri-cli/README.md

```diff
@@ -0,0 +1,145 @@
+
+**Troubleshooting:** If you have problems with migration, you can exclude some tasks with the configuration by setting the `false` flag (see `kol-select`).
+
+```json
+{
+	"migrate": {
+		"tasks": {
+			"kol-abbr-rename-property-_title-to-_label": true,
+			"kol-accordion-rename-property-_heading-to-_label": true,
+			"kol-badge-rename-property-_icon-only-to-_hide-label": true,
+			"kol-badge-remove-property-_hide-label": true,
+			"kol-badge-remove-property-_icon-only": true,
+			"kol-breadcrumb-rename-property-_aria-label-to-_label": true,
+			"kol-button-link-remove-property-_aria-current": true,
+			"kol-button-link-remove-property-_aria-label": true,
+			"kol-button-link-rename-property-_icon-only-to-_hide-label": true,
+			"kol-button-remove-property-_aria-current": true,
+			"kol-button-remove-property-_aria-label": true,
+			"kol-button-rename-property-_icon-only-to-_hide-label": true,
+			"kol-card-rename-property-_heading-to-_label": true,
+			"kol-card-rename-property-_headline-to-_label": true,
+			"kol-details-rename-property-_summary-to-_label": true,
+			"kol-icon-remove-property-_part": true,
+			"kol-icon-rename-property-_aria-label-to-_label": true,
+			"kol-checkbox-rename-property-_type-to-_variant": true,
+			"kol-color-rename-property-_list-to-_suggestions": true,
+			"kol-date-rename-property-_list-to-_suggestions": true,
+			"kol-input-email-rename-property-_list-to-_suggestions": true,
+			"kol-input-number-rename-property-_list-to-_suggestions": true,
+			"kol-input-radio-rename-property-_list-to-_options": true,
+			"kol-input-range-rename-property-_list-to-_suggestions": true,
+			"kol-input-text-rename-property-_list-to-_suggestions": true,
+			"kol-link-group-rename-property-_aria-label-to-_label": true,
+			"kol-link-group-remove-property-_ordered": true,
+			"kol-nav-rename-property-_aria-label-to-_label": true,
+			"kol-nav-rename-property-_compact-to-_hide-label": true,
+			"kol-nav-remove-property-_has-compact-button": true,
+			"kol-pagination-rename-property-_count-to-_total": true,
+			"kol-progress-rename-property-_type-to-_variant": true,
+			"kol-quote-rename-property-_caption-to-_label": true,
+			"kol-select-rename-property-_height-to-_rows": false,
+			"kol-select-rename-property-_list-to-_options": false,
+			"kol-skip-nav-rename-property-_aria-label-to-_label": true,
+			"kol-span-rename-property-_icon-only-to-_hide-label": true,
+			"kol-split-button-remove-property-_aria-label": true,
+			"kol-table-rename-property-_caption-to-_label": true,
+			"kol-tabs-rename-property-_aria-label-to-_label": true,
+			"kol-tabs-rename-property-_tab-align-to-_align": true,
+			"kol-toast-rename-property-_heading-to-_label": true,
+			"kol-version-rename-property-_version-to-_label": true
+		}
+	}
+}
+```
+
+## Troubleshooting
+
+If the migration failed, you can reset the migration with `git reset --hard HEAD~1`.
```

Comment:
Ich würde es noch erweitern – man muss nicht alles wegschmeißen, nur weil eine Datei falsch migriert wurde:

"If the migration failed, you can reset the migration with `git reset --hard HEAD~1` or by discarding the affected files."